src/web-components/KSLHighlightElement.ts

```diff
@@ -55,7 +56,7 @@ const templateHTML = `
     :host(:hover),
     :host([selected]) {
       border-color: var(--ksl-color-primary, ${Colors.Primary});
-      z-index: 20;
+      z-index: var(--ksl-z-index, 9998);
```

Comment:
This value is much higher than before, and this former 20 and below 10 are now the same, is that on purpose?

Reply:
I agree, I forget to keep same value ration